### PR TITLE
Explicitly set the test requirements in requirements-test.in

### PR DESCRIPTION
**Pull request by mprahl**
Using two separate files to generate a compiled requirements file confuses dependabot.

### CODE REVIEW

**Comment by mprahl** [comment]
@yashvardhannanavati @lcarva if we want to keep dependabot enabled and it to be useful, I believe we will need this change.

If we decided against it, I can disable dependabot.

File: requirements-test.in [hunk 1]
```diff
@@ -1,3 +1,11 @@
+celery
```
Comment (yashvardhannanavati) [inline requirements-test.in:1]: Can we please add a comment here explaining this change.

**Comment by yashvardhannanavati** [comment]
Just a single change. Looks great! Thanks Matt!

**Comment by yashvardhannanavati** [comment]
Please feel free to merge it after the comment is addressed :)

**Pull request merged**